src/keyboard.rs

```diff
@@ -0,0 +1,1207 @@
+    Copy,
+    /// Found on Sun’s USB keyboard.
+    Cut,
+    /// Found on Sun’s USB keyboard.
+    Find,
+    /// Found on Sun’s USB keyboard.
+    Open,
+    /// Found on Sun’s USB keyboard.
+    Paste,
+    /// Found on Sun’s USB keyboard.
+    Props,
+    /// Found on Sun’s USB keyboard.
+    Select,
+    /// Found on Sun’s USB keyboard.
+    Undo,
+    /// Use for dedicated <kbd>ひらがな</kbd> key found on some Japanese word processing keyboards.
+    Hiragana,
+    /// Use for dedicated <kbd>カタカナ</kbd> key found on some Japanese word processing keyboards.
+    Katakana,
+    /// <kbd>F</kbd>
+    F1,
+    /// <kbd>F</kbd>
+    F2,
+    /// <kbd>F</kbd>
+    F3,
+    /// <kbd>F</kbd>
+    F4,
+    /// <kbd>F</kbd>
+    F5,
+    /// <kbd>F</kbd>
+    F6,
+    /// <kbd>F</kbd>
+    F7,
+    /// <kbd>F</kbd>
+    F8,
+    /// <kbd>F</kbd>
+    F9,
+    /// <kbd>F10</kbd>
+    F10,
+    /// <kbd>F11</kbd>
+    F11,
+    /// <kbd>F12</kbd>
+    F12,
+    /// <kbd>F13</kbd>
+    F13,
+    /// <kbd>F14</kbd>
+    F14,
+    /// <kbd>F15</kbd>
+    F15,
+    /// <kbd>F16</kbd>
+    F16,
+    /// <kbd>F17</kbd>
+    F17,
+    /// <kbd>F18</kbd>
+    F18,
+    /// <kbd>F19</kbd>
+    F19,
+    /// <kbd>F20</kbd>
+    F20,
+    /// <kbd>F21</kbd>
+    F21,
+    /// <kbd>F22</kbd>
+    F22,
+    /// <kbd>F23</kbd>
+    F23,
+    /// <kbd>F24</kbd>
+    F24,
+    /// <kbd>F25</kbd>
+    F25,
+    /// <kbd>F26</kbd>
+    F26,
+    /// <kbd>F27</kbd>
+    F27,
+    /// <kbd>F28</kbd>
+    F28,
+    /// <kbd>F29</kbd>
+    F29,
+    /// <kbd>F30</kbd>
+    F30,
+    /// <kbd>F31</kbd>
+    F31,
+    /// <kbd>F32</kbd>
+    F32,
+    /// <kbd>F33</kbd>
+    F33,
+    /// <kbd>F34</kbd>
+    F34,
+    /// <kbd>F35</kbd>
+    F35,
+}
+
+/// Key represents the meaning of a keypress.
+///
+/// Specification:
+/// https://w3c.github.io/uievents-key/
+#[non_exhaustive]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum Key<'a> {
+    /// A key string that corresponds to the character typed by the user,
+    /// taking into account the user’s current locale setting, modifier state,
+    /// and any system-level keyboard mapping overrides that are in effect.
+    Character(&'a str),
+
+    /// This variant is used when the key cannot be translated to any
+    /// other variant.
+    ///
+    /// The native scancode is provided (if available) in order
+    /// to allow the user to specify keybindings for keys which
+    /// are not defined by this API.
+    Unidentified(NativeKeyCode),
+
+    /// Contains the text representation of the dead-key
+    /// when available.
+    ///
+    /// ## Platform-specific
+    /// - **Web:** Always contains `None`
+    Dead(Option<char>),
+
+    /// The `Alt` (Alternative) key.<br> This key enables the alternate modifier function for interpreting concurrent or subsequent keyboard input.<br> This key value is also used for the Apple `Option` key.
```

Comment:
You should probably use actual line breaks rather than manually putting in a `<br>`. The documentation should be readable in the source code too.

Reply:
Oops. You are completely right.